src/common/util.js

```diff
@@ -108,6 +138,12 @@ function skipTrailingComment(text, index) {
 // This one doesn't use the above helper function because it wants to
 // test \r\n in order and `skip` doesn't support ordering and we only
 // want to skip one newline. It's simple to implement.
+/**
```

Comment:
Should this comment be part of the JSDoc? Same question for some comments further down.

Reply:
This comment -> I think no
Because it is really an internal comment
Or in other words: When I want to call this function, I did not expect the content of this comment